.travis.yml

```diff
@@ -1,7 +1,7 @@
 sudo : false
 language: node_js
 node_js:
-  - "4.2"
+  - "6.9.1"
```

Comment:
Are you requiring this _exact_ version of node for this? Not sure if `>=6.9.1` works in this syntax